sklearn/linear_model/_base.py

```diff
@@ -33,14 +33,14 @@
 from ..utils.validation import _deprecate_positional_args
 from ..utils import check_random_state
 from ..utils.extmath import safe_sparse_dot
+from ..utils.extmath import _incremental_weighted_mean_and_var
```

Comment:
I think that this function should not be available anymore in the `main` branch?

Reply:
It is the reported error with the pytest collection failure. You can look at the changes: https://github.com/scikit-learn/scikit-learn/pull/19422